cpp/unreal_projects/SpearSim/Config/DefaultEngine.ini

```diff
@@ -27,8 +27,8 @@ r.EyeAdaptation.ExponentialTransitionDistance=0
 r.Streaming.FullyLoadUsedTextures=1
 
 [/Script/EngineSettings.GameMapsSettings]
-GameDefaultMap=/Game/DefaultMap.DefaultMap
-EditorStartupMap=/Game/DefaultMap.DefaultMap
+GameDefaultMap=/Game/Scenes/default/DefaultMap.DefaultMap
```

Comment:
`/Game/Scenes/default/Maps/DefaultMap.DefaultMap` to match the convention we established in our previous discussion.

Reply:
Done